javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,11 @@
+const dogYearOfBirth = 1985;
+let dogYearFuture = 2027;
+let shouldShowResultInDogYears = true;
+if (shouldShowResultInDogYears === true){
+    let dogYear = (dogYearFuture-dogYearOfBirth)*7;
+ console.log("Your dog will be " + dogYear + " dog years old in" + dogYearFuture);   
+}
+else if(shouldShowResultInDogYears === false) {
+    let humanYear = dogYearFuture-dogYearOfBirth;
+    console.log("Your dog will be " + humanYear + " human years old in" + dogYearFuture);
```

Comment:
here you can replace humanYear with dogYear

Reply:
done. thank you.